fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -187,35 +186,26 @@ void init() throws IOException {
     }
 
     @Override
-    public void writeTo(final RdfStream rdfStream, final Class<?> type,
+    public void writeTo(final RdfNamespacedStream nsStream, final Class<?> type,
                         final Type genericType, final Annotation[] annotations,
                         final MediaType mediaType,
                         final MultivaluedMap<String, Object> httpHeaders,
                         final OutputStream entityStream) throws IOException {
 
-        try {
-            final RdfStream nsRdfStream = new NamespaceRdfContext(rdfStream.session());
-
-            rdfStream.namespaces(nsRdfStream.namespaces());
-
-            final Node subject = VIEW_HELPERS.getContentNode(rdfStream.topic());
-
-            final Model model = rdfStream.asModel();
+        final Node subject = VIEW_HELPERS.getContentNode(nsStream.stream.topic());
```

Comment:
You are correct. I was misled by the simple signature on the `RdfStream` interface:
https://github.com/fcrepo4/fcrepo4/pull/962/files#diff-890f544f149c4ed8a3017db0cc81846bR29

..overlooking the fact that it extends another, larger interface.